etna/packages/etna-js/selectors/maybe.ts

```diff
@@ -0,0 +1,66 @@
+export type Maybe<T> = null | [T];
```

Comment:
Because some of these methods are used in the data frame utils, moving them to `etna-js`